base/ltfilehook.dtx

```diff
@@ -30,15 +30,16 @@
 %
 %%% From File: ltfilehook.dtx
 %
-%    \begin{macrocode}
-\providecommand\ltfilehookversion{v1.0o}
-\providecommand\ltfilehookdate{2024/02/04}
-%    \end{macrocode}
-%
+%<structuredlog>\providecommand\ltfilehookdate{2024/02/11}
+%<structuredlog>\providecommand\ltfilehookversion{v1.0o}
 %<*driver>
-
+% \fi
+\ProvidesFile{ltfilehook.dtx}
+             [2024/02/11 v1.0o LaTeX Kernel (hooks)]
```

Comment:
I don't think we want to have to specify the date in more than one place -- or am I reading this diff wrong?

Reply:
I wondered if you'd say that:-)  I ended up with duplicated dates in two files (ltfilehook and ltluatex)  it may be possible to avoid it by adjusting the docstrip guards, or as you say we could keep the macros, I'll look again.

Reply:
re-arranged to use \ltfilehookdate so date only given once